assets/wizards/dashboard/views/DevInfo/style.scss

```
@@ -0,0 +1,62 @@
+@import '../../../../shared/scss/colors';
+@import '~@wordpress/base-styles/colors';
+
+.newspack-dashboard__news {
+	details {
+		margin-bottom: 5px;
+		border: 1px solid $light-gray-200;
+		padding: 20px;
+		border-radius: 3px;
+		> div {
+			border-radius: 2px;
+			margin: 20px 0 0;
+			padding-left: 40px;
+			h3:first-child {
+				margin-top: 0;
+			}
+		}
+		ul {
+			list-style: disc;
+			padding-left: 18px;
+		}
+		li {
+			margin-bottom: 0;
+		}
+		ul:last-child {
+			margin-bottom: 0;
+		}
+		h3 {
```

I think it might be good to scale these down to match the body font (or a couple pixels larger); they seem a bit overwhelming to me compared to the summaries: 

```suggestion
		h3 {
			font-size: inherit;
```

---

Went with a slightly larger font size than the rest, so they still look header-y, in e2cd04c.